github/Requester.py

```diff
@@ -67,19 +67,86 @@
 atLeastPython3 = sys.hexversion >= 0x03000000
 
 
+class RequestsResponse:
+    # mimic the httplib response object
+    def __init__(self, r):
+        self.status = r.status_code
+        self.headers = r.headers
+        self.text = r.text
+
+    def getheaders(self):
+        return self.headers.iteritems()
```

Comment:
`iteritems()` is not available in Python3.

I am afraid we have to do something like:
```
if atLeastPython3:
	return self.headers.items()  # items() in Python3 returns an iterator
else:
	return self.headers.iteritems()
```

https://docs.python.org/3/library/stdtypes.html#dict.items